### DISCUSSION TITLE
Could i let sniprun's results diplay on toggleterm?🤔

### DISCUSSION THREAD

**Discussion by Lotfree618** [michaelb/sniprun#212] [Q&A] [answered]
Display on toggleterm,like vscode code-runner "run In Terminal ",so i can run codes with input.
Or is there any way to make the input code run.

**Comment by michaelb** [comment, marked as answer]
If by 'code with input' you mean you want to run a snippet like


```
a = int( input ("please input a number"))
print ("you chose: " + str(a))
```

it's not currently possible.

I might work someday, as you're not the only one wanting this feature, but I wouldn't expect it for tomorrow if I were in yours shoes
